include/deal.II/particles/particle_handler.h

```diff
@@ -205,10 +205,10 @@ namespace Particles
      * @note While this function is used in step-19, it is not an efficient
      *   function to use if the number of particles is large. That is because
      *   to find the particles that are located in one cell costs
-     *   ${\cal O)(\log N)$ where $N$ is the number of overall particles. Since
+     *   $(\cal O)(\log N)$ where $N$ is the number of overall particles. Since
```

Comment:
We need curly brackets around `\cal O`.
See https://arachnoid.com/latex/?equ=%24%7B%5Ccal%20O%7D(%5Clog%20N)%24
```suggestion
     *   ${\cal O}(\log N)$ where $N$ is the number of overall particles. Since
```